misc/xml-to-record-converter/src/test/resources/xml/sample_26.xml

```diff
@@ -0,0 +1,5 @@
+<bk:book xmlns:bk="http://example.com/book" xmlns:au="http://example.com/author" xmlns:da="http://example.com/date">
+    <bk:title>string</bk:title>
+    <au:title>string</au:title>
+    <da:title>string</da:title>
```

Comment:
In this example or separate example, shall we add other fields different than `title` to validate the behaviour

Reply:
Added more tests